src/graphql/queries/EntryValues.ts

```diff
@@ -1,22 +1,165 @@
 import { gql } from 'react-apollo';
 import { decode } from '../../services/UrlStringCreator';
+import { ComponentConfig } from '../../typings/schema';
+import { DataSets, checkTypes, Query } from '../../typings/schema';
 
-const QUERY_ENTRY_VALUES = ({ match, currentCollection, values }) => {
-    values = values || [];
+// `type: never` makes the type checker report an error if the case switch does not handle all types
+function checkUnknownComponent(type: never) {
+    console.error(`Type ${(type as ComponentConfig).type} is not handled!`);
+}
+
+function getTitleProp(typeId: string, otherCollections: Array<{collectionId: string, summaryProperties: {title?: {value: string}}}>): string {
+    for (let i = 0; i < otherCollections.length; i++) {
+        if (otherCollections[i].collectionId === typeId) {
+            const title = otherCollections[i].summaryProperties.title;
+            if (title) {
+                return '.' + title.value;
+            } else {
+                return '.uri';
+            }
+        }
+    }
+    return '.uri';
+}
+
+export function makeDefaultViewConfig(properties: Array<{ isList: boolean, isValueType: boolean, name: string, referencedCollections: { items: string[] } }>, summaryProperties: { title?: { value: string } }, otherCollections: Array<{ collectionId: string, summaryProperties: { title?: { value: string } } }>): Array<ComponentConfig> {
+    const title: ComponentConfig[] = summaryProperties.title ?
+        [{ type: 'TITLE', formatter: [], subComponents: [{ type: 'PATH', formatter: [], value: summaryProperties.title.value }] }] :
```

Comment:
Add a ```// todo: "implement single-component-config-create" function, when that is created for the new components in the view config``` here.

Like the todo says, we need a function that returns an empty or partially filled componentConfig for the view config. When we created that, we should not forget to also let this utilize that.